### DISCUSSION TITLE
global custom *.d.ts files and check unavaiable, miss some config? eslint or tsc check

### DISCUSSION THREAD

**Discussion by codingwesley** [eslint/eslint#17758] [Q&A] [answered]
```ts
    interface TimelineContext {
      timelineId: strng;
      programId: string;
    }
```

example code, have `strng` not is right, and not have warning, some one help me

**Comment by Rec0iL99** [comment, marked as answer]
Hi @codingwesley, that is a syntax error and ESLint is not for finding syntax errors. I guess you can just restart your TS server and get hints to fix your syntax error. If you have more questions on **linting** typescript code please contact the `typescript-eslint` team for [help](https://discord.com/invite/FSxKq8Tdyg).

**Reply by codingwesley** [reply]
Thanks, I found is TS server throw error in ts file, and *.d.ts file not working.
